### DISCUSSION TITLE
Script section GONE....

### DISCUSSION THREAD

**Discussion by francescoadriani** [arendst/Tasmota#14615] [Q&A] [answered]
This is my user_config_override...
Where i'm wrong?

![immagine_2022-01-26_224045](https://user-images.githubusercontent.com/90975508/151251592-07e0ce17-2d45-401c-883d-07d913bfc777.png)

#ifndef _USER_CONFIG_OVERRIDE_H_
  #define _USER_CONFIG_OVERRIDE_H_

  #ifdef PROJECT
  #undef PROJECT
  #endif
  #define PROJECT                "DispositivoInCasaRice_tasmota"

  #ifdef BOOT_LOOP_OFFSET
  #undef BOOT_LOOP_OFFSET
  #endif
  #define BOOT_LOOP_OFFSET       0                 // [SetOption36] Number of boot loops before starting restoring defaults (0 = disable, 1..200 = boot loops offset)

  // -- Wifi ----------------------------------------

  #ifdef WIFI_GATEWAY
  #undef WIFI_GATEWAY
  #endif
  #define WIFI_GATEWAY           "10.2.232.1"     // [IpAddress2] If not using DHCP set Gateway IP address

  #ifdef WIFI_SUBNETMASK
  #undef WIFI_SUBNETMASK
  #endif
  #define WIFI_SUBNETMASK        "255.255.248.0"   // [IpAddress3] If not using DHCP set Network mask

  #ifdef WIFI_DNS
  #undef WIFI_DNS
  #endif
  #define WIFI_DNS               "10.2.232.1"     // [IpAddress4] If not using DHCP set DNS IP address (might be equal to WIFI_GATEWAY)

  #ifdef STA_SSID1
  #undef STA_SSID1
  #endif
  #define STA_SSID1              "xxxxxxxxxxxx"                // [Ssid1] Wifi SSID

  #ifdef STA_PASS1
  #undef STA_PASS1
  #endif
  #define STA_PASS1              "xxxxxxxxx"                // [Password1] Wifi password

  #ifdef STA_SSID2
  #undef STA_SSID2
  #endif
  #define STA_SSID2              "xxxxxxxxxxxxx"                // [Ssid2] Optional alternate AP Wifi SSID

  #ifdef STA_PASS2
  #undef STA_PASS2
  #endif
  #define STA_PASS2              "xxxxxxxxxxxxxxx"                // [Password2] Optional alternate AP Wifi password

  #ifdef WIFI_ARP_INTERVAL
  #undef WIFI_ARP_INTERVAL
  #endif
  #define WIFI_ARP_INTERVAL      0                 // [SetOption41] Send gratuitous ARP interval

  #ifdef MQTT_HOST
  #undef MQTT_HOST
  #endif
  #define MQTT_HOST              "10.2.233.2"

  #ifdef MQTT_FULLTOPIC
  #undef MQTT_FULLTOPIC
  #endif
  #define MQTT_FULLTOPIC         "%topic%/%prefix%/"

  #ifdef TELE_PERIOD
  #undef TELE_PERIOD
  #endif
  #define TELE_PERIOD            60 

  #ifdef MQTT_INDEX_SEPARATOR
  #undef MQTT_INDEX_SEPARATOR
  #endif
  #define MQTT_INDEX_SEPARATOR   true

  #ifdef FRIENDLY_NAME
  #undef FRIENDLY_NAME
  #endif
  #define FRIENDLY_NAME          "xxxxxxxxxxxxx"

  #ifdef GUI_SHOW_HOSTNAME
  #undef GUI_SHOW_HOSTNAME
  #endif
  #define GUI_SHOW_HOSTNAME   true

  // -- MAIL ----------------------------------------
  #ifndef ESP32
  #ifndef USE_SENDMAIL
  #define USE_SENDMAIL
  #endif
  #endif

  #ifdef EMAIL_SERVER
  #undef EMAIL_SERVER
  #endif
  #define EMAIL_SERVER                "xxxxxxxxxxxxxx"

  #ifdef EMAIL_PORT
  #undef EMAIL_PORT
  #endif
  #define EMAIL_PORT                  465

  #ifdef EMAIL_USER
  #undef EMAIL_USER
  #endif
  #define EMAIL_USER                  "xxxxxxxxxxxxx"

  #ifdef EMAIL_PASSWORD
  #undef EMAIL_PASSWORD
  #endif
  #define EMAIL_PASSWORD              "xxxxxxxxxxxxxxxxxx"

  #ifdef EMAIL_FROM
  #undef EMAIL_FROM
  #endif
  #define EMAIL_FROM                  "xxxxxxxxxxxx"

  #ifdef LATITUDE
  #undef LATITUDE
  #endif
  #define LATITUDE               43.3710192         // [Latitude] Your location to be used with sunrise and sunset

  #ifdef LONGITUDE
  #undef LONGITUDE
  #endif
  #define LONGITUDE              12.2433589    

  #ifdef MY_LANGUAGE
  #undef MY_LANGUAGE
  #endif
  #define MY_LANGUAGE            it_IT 

  #ifdef USE_DOMOTICZ
  #undef USE_DOMOTICZ
  #endif

  #ifdef USE_HOME_ASSISTANT
  #undef USE_HOME_ASSISTANT
  #endif

  #ifndef USE_WEBSERVER
  #define USE_WEBSERVER
  #endif

  #ifdef WEB_PORT
  #undef WEB_PORT
  #endif
  #define WEB_PORT            8081 

  #ifndef USE_PING
  #define USE_PING
  #endif

  #ifdef USE_RULES
  #undef USE_RULES
  #endif

  #ifndef USE_SCRIPT
  #define USE_SCRIPT  
  #endif                             // Add support for script (+17k code)
      #ifndef LARGE_ARRAYS
      #define LARGE_ARRAYS
      #endif
      #ifndef USE_SCRIPT_WEB_DISPLAY
      #define USE_SCRIPT_WEB_DISPLAY
      #endif
      #ifndef SCRIPT_FULL_WEBPAGE
      #define SCRIPT_FULL_WEBPAGE
      #endif
      #ifndef USE_SCRIPT_TIMER
      #define USE_SCRIPT_TIMER	
      #endif		
      #ifndef USE_SCRIPT_GLOBVARS
      #define USE_SCRIPT_GLOBVARS  
      #endif
      #ifndef USE_GOOGLE_CHARTS
      #define USE_GOOGLE_CHARTS
      #endif
      #ifndef USE_DSIPLAY_DUMP
      #define USE_DSIPLAY_DUMP
      #endif
      #ifdef USE_DSIPLAY_DUMP
      #undef USE_DSIPLAY_DUMP                    // Script: Add FAT FileSystem Support
      #endif
      #ifndef SUPPORT_MQTT_EVENT
      #define SUPPORT_MQTT_EVENT                     // Support trigger event with MQTT subscriptions (+3k5 code)
      #endif

  #ifndef USE_DISPLAY
  #define USE_DISPLAY  
  #endif

  #ifndef USE_SR04
  #define USE_SR04
  #endif

  //#define USE_ETHERNET                             // Add support for ethernet (Currently fixed for Olimex ESP32-PoE)
    #ifndef USE_WT32_ETH01
    #define USE_WT32_ETH01                    // Script: Add FAT FileSystem Support
    #endif

    #ifdef ETH_TYPE
    #undef ETH_TYPE                    // Script: Add FAT FileSystem Support
    #endif
    #define ETH_TYPE          0                    // [EthType] 0 = ETH_PHY_LAN8720, 1 = ETH_PHY_TLK110, 2 = ETH_PHY_IP101
    
    #ifdef ETH_ADDRESS
    #undef ETH_ADDRESS                    // Script: Add FAT FileSystem Support
    #endif
    #define ETH_ADDRESS       1                    // [EthAddress] 0 = PHY0 .. 31 = PHY31
    
    #ifdef ETH_CLKMODE
    #undef ETH_CLKMODE                    // Script: Add FAT FileSystem Support
    #endif
    #define ETH_CLKMODE       0   

#endif  // _USER_CONFIG_OVERRIDE_H_

**Comment by barbudor** [comment, marked as answer]
The Script editor is in the Console page, not in the Configuration page

**Comment by francescoadriani** [comment]
Wow, it's moved! Ok it's better then before
![image](https://user-images.githubusercontent.com/90975508/151254370-b88bcdde-7965-4e55-b28a-f69fd482b101.png)
